pypeclub/OpenPype
# TrayPublisher: added persisting of last selected project

## Brief description
Last selected project is preselected in Tray UI to make selection quicker.

## Additional info
Last selected project is stored in .json in app folder - eg. next to unzipped version zip.


## Testing notes:
1. open Tray Publisher and select project
2. close Tray Publisher - you don't need to publish anything
3. open TP again - last project should be preselected
